docker/config/config.go

```diff
@@ -103,7 +107,7 @@ func Load(fileName string) (*Config, error) {
 		}
 
 		authenticationToken := string(b)
-		usernameAndPassword := strings.Split(authenticationToken, ":")
+		usernameAndPassword := strings.SplitN(authenticationToken, ":", 2)
```

Comment:
This was the initial change I made in order to fix the decoding of the base64 data I gleaned from my `~/.docker/config.json`. The commit message in e33c4e28ec51fbac4d319ee050aad291448a656a discusses how I replicated and tested this:

```
The shape of the moving parts (username, password) was largely reverse
engineered by:

1. Generating a service account with gcr access and saving to `key.json`.
2. `cat key.json | docker login -u _json_key --password-stdin https://gcr.io`
3. `cat `~/.docker/config.json` should now contain somethin like:```
    "gcr.io" : {
      "auth" : "..."
      }
```